### PR TITLE
test on all supported ruby versions

### PR DESCRIPTION
**Pull request by grosser**
picking 2.2+ since everything else is end-of-life ...

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
 language: ruby
 rvm:
+ - 2.2
+ - 2.2
```
Comment (ddnexus) [inline .travis.yml]: duplicate

Comment (grosser) [reply .travis.yml]: updated

**Comment by ddnexus** [comment]
So is this failing because of the missing #31?

**Comment by grosser** [comment]
yeah ... so 2.2 is not supported because it's using `Hash#dig` I'll combine them here for less madness

**Comment by grosser** [comment]
ready!

**Pull request merged**